1.0.3/specifications.md

```diff
@@ -19,8 +20,8 @@ abstract:
   format for [CodeMeta](https://codemeta.github.io/) [JSON](https://github.com/codemeta/codemeta/blob/master/codemeta.json) files.
 geometry: margin=2cm
 version: 1.0.3
-specs-version: 1.0.3-3
-release-date: 29 Aug 2018
+specs-version: 1.0.3-4
+release-date: 22 Oct 2019
```

Comment:
we could update this date to the date when the button is pressed

Reply:
```suggestion
release-date: 04 Nov 2019
```